Pilot/pilotTools.py

```diff
@@ -687,7 +687,7 @@ def cancelTimer(self):
             self._timer.cancel()
 
 
-def sendMessage(url, pilotUUID, wnVO, method, rawMessage):
+def sendMessage(url, pilotUUID, wnVO, method, rawMessage, withJWT=False, jwt={}):
```

Comment:
You can do it without the boolean flag

```suggestion
def sendMessage(url, pilotUUID, wnVO, method, rawMessage, jwt={}):
```

and later simply check `if jwt`